src/files-card-body.css

```diff
@@ -167,7 +168,7 @@
     &.view-details .item-name a::before {
         display: inline-block;
         margin-inline-end: var(--pf-v5-global--spacer--sm);
-        // FIXME: Figure out why there's an offset
+        /* FIXME: Figure out why there's an offset */
```

Comment:
:sweat_smile: 

We never did figure this out. These kinds of things are usually related to line-height or alignment being computed weirdly, or specific font metrics causing odd stuff.

This works and is tested, and probably related to something deep in PatternFly. It might need to be re-evaluated with PF6. (Or might not.)

If anything is off with PF6 overall, we should probably look at the FIXMEs like this too.